### PR TITLE
Add product tour

### PR DESCRIPTION
**Pull request by fabrice-adamik**
Add new component (firstvisittooltip) to catch user's attention on the toggle.
Display the tooltip right after the WelcomeModal
tooltip isn't visible after user aknowledges it
Update welcome modal picture, text, and video

https://github.com/user-attachments/assets/06604fbf-1459-4b54-9022-64884021dd32

### CODE REVIEW

File: public/intro.svg
Comment (hakim-adamik) [inline public/intro.svg:1]: The file seems really heavy for an SVG, it should be possible to make it a lot lighter?

File: src/app/wallets/WalletModalTrigger.tsx
Comment (hakim-adamik) [inline src/app/wallets/WalletModalTrigger.tsx:1]: This should remain here, it's useful for Next.js

File: src/components/ui/firstvisittooltip.tsx
Comment (hakim-adamik) [inline src/components/ui/firstvisittooltip.tsx:1]: To be confirmed with @henri-ly but I think everything in `/src/components/ui/` is meant to remain non-specific, i.e a library of generic components that can be used anywhere in the app.

For now I would just move this file 1 level up, into `/src/components/` and also rename it `FirstVisitTooltip.tsx` to respect naming conventions.

File: src/components/ui/firstvisittooltip.tsx
Comment (hakim-adamik) [inline src/components/ui/firstvisittooltip.tsx:28]: `console.log` are good for debugging :slightly_smiling_face: but we shouldn't keep them in production, especially on client side as they are all visible in the browser's console.

I think all the ones in this file can be removed.

**Review by hakim-adamik — COMMENTED** [review]
I'll let @henri-ly review as well

File: public/intro.svg
Comment (fabrice-adamik) [reply public/intro.svg:1]: fixed using PNG instead

File: src/components/ui/firstvisittooltip.tsx
Comment (fabrice-adamik) [reply src/components/ui/firstvisittooltip.tsx:1]: Let's wait for @henri-ly's feedback on this.

Comment (henri-ly) [reply src/components/ui/firstvisittooltip.tsx:1]: Yeah totally this, /src/component/ui is something generate by https://ui.shadcn.com/ and we want to reuse across the app

**Review by henri-ly — APPROVED** [review]

**Pull request merged**
